pkg/services/auth/auth_token.go

```diff
@@ -404,6 +410,35 @@ func (s *UserAuthTokenService) rotatedAfterParam() int64 {
 	return getTime().Add(-s.Cfg.LoginMaxInactiveLifetime).Unix()
 }
 
+const (
+	concurrentSessionLimit    = 3
+	concurrentSessionLimitKey = "concurrentSessionLimit"
+)
+
+func (s *UserAuthTokenService) isConcurrentSessionLimitEnabled() bool {
+	return s.Cfg.FeatureToggles[concurrentSessionLimitKey]
```

Comment:
Why feature toggle and not a setting in [[auth]](https://grafana.com/docs/grafana/latest/administration/configuration/#auth)?

Reply:
I'm not really sure about this. Maybe @mjseaman can give us with more details.
The issue definition specifies it as such, I suppose it's somehow related with AWS requirements.

Reply:
Feature toggles are only used for temporary features that eventually will be enabled/included per default. Usually used when developing new experimental features. Adding it as a separate setting is basically a toggle.

Reply:
Agreed with @marefr, we don't need feature toggle in this case, this is not an experimental feature.

Reply:
Hi @marefr and @alexanderzobnin, this is a licensing enforcement feature, which we are making available to AWS before rolling it out ourselves. 

The purpose of the feature is to discourage multiple users from signing in using the same credentials (and therefore paying for fewer users). I believe that if the concurrent session limit is a setting, then customers will always be able to disable the limit, right? That would make our license enforcement ineffective. 

The plan is to let AWS roll this out for their managed service in November, begin measuring the number of concurrent sessions for our own users, and then carefully roll it out as a default, non-configurable part of our own paid products.